commit e506cb23cfce35d1bc997d1e280f4dc40c9b3397
Author: Severin Gehwolf <sgehwolf@openjdk.org>
Date:   Mon Aug 16 09:57:28 2021 +0000

    8272332: --with-harfbuzz=system doesn't add -lharfbuzz after JDK-8255790
    
    Backport-of: d38b31438dd4730ee2149c02277d60c35b9d7d81

diff --git openjdk.orig/make/modules/java.desktop/lib/Awt2dLibraries.gmk openjdk/make/modules/java.desktop/lib/Awt2dLibraries.gmk
index 4d0c0c00dbf..ef7eadae206 100644
--- openjdk.orig/make/modules/java.desktop/lib/Awt2dLibraries.gmk
+++ openjdk/make/modules/java.desktop/lib/Awt2dLibraries.gmk
@@ -435,7 +435,7 @@ endif
 
 ifeq ($(USE_EXTERNAL_HARFBUZZ), true)
    LIBFONTMANAGER_EXTRA_SRC =
-   BUILD_LIBFONTMANAGER_FONTLIB += $(LIBHARFBUZZ_LIBS)
+   BUILD_LIBFONTMANAGER_FONTLIB += $(HARFBUZZ_LIBS)
 else
    LIBFONTMANAGER_EXTRA_SRC = libharfbuzz
 
